src/postgres/commands/createPostgresFunctionQuery.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizard } from "vscode-azureextensionui";
+import { nonNullProp } from "../../utils/nonNull";
+import * as vscodeUtil from '../../utils/vscodeUtils';
+import { PostgresDatabaseTreeItem } from "../tree/PostgresDatabaseTreeItem";
+import { connectPostgresDatabase } from "./connectPostgresDatabase";
+import { FunctionQueryCreateStep } from "./FunctionQueryWizard/FunctionQueryCreateStep";
+import { FunctionQueryNameStep } from "./FunctionQueryWizard/FunctionQueryNameStep";
+import { FunctionQueryReturnTypeStep } from "./FunctionQueryWizard/FunctionQueryReturnTypeStep";
+import { IPostgresFunctionQueryWizardContext } from "./FunctionQueryWizard/IPostgresFunctionQueryWizardContext";
+
+export async function createPostgresFunctionQuery(wizardContext: IPostgresFunctionQueryWizardContext, treeItem?: PostgresDatabaseTreeItem): Promise<void> {
+    const wizard = new AzureWizard(wizardContext, {
+        promptSteps: [new FunctionQueryNameStep(), new FunctionQueryReturnTypeStep()],
+        executeSteps: [new FunctionQueryCreateStep()],
+        title: 'Create PostgreSQL Function Query'
+    });
+
+    await wizard.prompt();
+    await wizard.execute();
+    await vscodeUtil.showNewFile(nonNullProp(wizardContext, 'query'), 'query', '.sql');
+
+    if (treeItem) {
+        await connectPostgresDatabase(wizardContext, treeItem);
```

Comment:
Isn't the entrypoint on the "Functions" tree item now and not the database tree item?

Reply:
Yeah I forgot to change that